DistroLauncher/DistroLauncher.rc

```diff
@@ -46,12 +76,13 @@ END
 
 /////////////////////////////////////////////////////////////////////////////
 //
-// Icon
+// String Table
 //
 
-// Icon with lowest ID value placed first to ensure application icon
-// remains consistent on all systems.
-IDI_ICON1               ICON                    ".\images\icon.ico"
+STRINGTABLE
```

Comment:
One question. What happens with the Icon? I see it is deleted from the resource file